sdk/spring/scripts/get_suitable_versions_for_sb3.py

```diff
@@ -20,7 +20,7 @@
 import argparse
 import requests
 from log import log
-from sync_external_dependencies import version_bigger_than
+from sync_external_dependencies import version_greater_than
```

Comment:
nit: I think the version comparing should in a separate file, instead in the `sync_external_dependencies` file.